object_detection/scripts/get_objects.py

```diff
@@ -0,0 +1,116 @@
+"""
+"""
+
```

Comment:
Add a doctring here with the description of the script, parameters and return values/console prints